upbound_uxp/assets/dashboards/upbound_uxp_min_set.json

```diff
@@ -0,0 +1 @@
k Duration Seconds Count","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"formulas":[{"formula":"query1"}],"queries":[{"name":"query1","data_source":"metrics","query":"avg:uxp.workqueue_work_duration_seconds_bucket.count{*} by {host,pod,name,le}"}],"response_format":"timeseries","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}]},"layout":{"x":4,"y":8,"width":4,"height":2}},{"id":6712236079650739,"definition":{"title":"Workqueue Work Duration Seconds Sum","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"formulas":[{"formula":"query1"}],"queries":[{"name":"query1","data_source":"metrics","query":"sum:uxp.workqueue_work_duration_seconds_sum{*} by {host,pod,name}.as_count()"}],"response_format":"timeseries","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}]},"layout":{"x":8,"y":8,"width":4,"height":2}},{"id":3667451147019795,"definition":{"title":"Upjet Terraform Active CLI Invocations","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"formulas":[{"formula":"per_minute(query1)"}],"queries":[{"name":"query1","data_source":"metrics","query":"max:uxp.upjet_terraform_active_cli_invocations{*}"}],"response_format":"timeseries","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}]},"layout":{"x":0,"y":10,"width":4,"height":2}},{"id":4282148986424876,"definition":{"title":"Upjet Terraform Running Processes","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"formulas":[{"formula":"per_minute(query1)"}],"queries":[{"name":"query1","data_source":"metrics","query":"max:uxp.upjet_terraform_running_processes{*}"}],"response_format":"timeseries","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}]},"layout":{"x":4,"y":10,"width":4,"height":2}},{"id":2395363743244600,"definition":{"title":"Controller Runtime Reconcile Errors","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"formulas":[{"formula":"per_minute(query1)"}],"queries":[{"name":"query1","data_source":"metrics","query":"max:uxp.controller_runtime_reconcile_errors_total{*}.as_count()"}],"response_format":"timeseries","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}]},"layout":{"x":8,"y":10,"width":4,"height":2}}],"template_variables":[],"layout_type":"ordered","notify_list":[],"reflow_type":"fixed"}
```

Comment:
I think you can remove the metric descriptions and names from the dashboard description, as those are covered in the widget titles. I'd recommend instead providing a summary of the type of data available in the dashboard, and linking to relevant documentation on Upbound. 

Also, please add the Upbound logo to the dashboard as well.

Reply:
Please advise where to add the dashboard logo. Thanks.

Reply:
Removed metric descriptions from dashboard descriptions. Provided brief overview of type of data available.

Added dashboard logos.